src/notebook/notebookFeature.ts

```diff
@@ -26,9 +26,6 @@ type JupyterNotebookMetadata = Partial<{
 }>
 
 function getNotebookMetadata(notebook: vscode.NotebookDocument): JupyterNotebookMetadata['metadata'] | undefined{
-    if (notebook.metadata && 'custom' in notebook.metadata){
```

Comment:
This old approach of storing metadata is also no longer required as VS Code has completely migrated to using the new format